crates/ruma-events-macros/src/lib.rs

```diff
@@ -36,16 +36,20 @@ mod event_parse;
 /// # Examples
 ///
 /// ```ignore
+/// # // HACK: This is "ignore" because of cyclical dependency drama.
 /// use ruma_events_macros::event_enum;
 ///
 /// event_enum! {
-///     name: AnyBarEvent, // `BarEvent` has to be a valid type at `::ruma_events::BarEvent`
+///     kind: ToDevice,
 ///     events: [
 ///         "m.any.event",
 ///         "m.other.event",
 ///     ]
 /// }
 /// ```
+/// (The argument to `kind` has to be a valid identifier for `<EventKind as Parse>::parse`)
+//// TODO: Change above (`<EventKind as Parse>::parse`) to [] after fully qualified syntax is supported:
+////  https://github.com/rust-lang/rust/issues/74563
```

Comment:
Why does this commend start with four `/`es?

Reply:
To leave it out of the actual doc, its more a comment on the documentation itself.
 
 https://doc.rust-lang.org/stable/reference/comments.html#doc-comments
 
 ```
    //   - Only a comment
    ///  - Outer line doc (exactly 3 slashes)
    //// - Only a comment
```

Reply:
I understand that that makes it not a doc comment but I've never seen four slashes as "comment on docs" before.

Reply:
Utilizing three, imo, felt more like a "lowering" from the doc, making four then is more like "above" the doc.